recipe/meta.yaml

```diff
@@ -26,11 +26,12 @@ build:
 
 requirements:
   build:
-    - python
     - setuptools
     - pip
+  host:
```

Comment:
Usually pure python projects don't have a build section, all the build things go in host

Reply:
Exactly. Already corrected in the merged version.